test/integration/sequelize.test.js

```diff
@@ -129,6 +129,7 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
               err.message.includes('invalid port number') ||
               err.message.match(/should be >=? 0 and < 65536/) ||
               err.message.includes('Login failed for user') ||
+              err.message.includes('SQL30081N') ||
```

Comment:
Do we have anything more digestible?

Reply:
The err.message would contain `SQL30081N A communication error has been detected.`. So, we can have `err.message.includes('SQL30081N A communication error has been detected') ||` here.

Reply:
Sounds good!